### PR TITLE
breaking: @cordova/eslint-config w/ applied fix

### PR DESCRIPTION
**Pull request by erisu**
### Motivation and Context

Use Cordova's ESLint Configs.

https://github.com/apache/cordova/issues/142

### Testing

- `npm t`

### Checklist

- [x] I've run the tests to see all new and existing tests pass

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,9 +18,9 @@
   ],
   "scripts": {
     "cover": "nyc jasmine",
-    "eslint": "eslint . \"bin/**/!(*.*)\"",
+    "lint": "eslint . \"bin/**/!(*.*|gitignore)\"",
```
Comment (timbru31) [inline package.json:21]: This differs from our others - can't we just use `eslint .`?

Comment (raphinesse) [reply package.json:21]: No, unfortunately not. The second argument glob is necessary to also lint Node.js "binaries" that do not have a `.js` extension.

**Comment by NiklasMerz** [comment]
Good to merge for major release?

**Comment by timbru31** [comment]
friendly ping. master is already at next major, IMHO good to merge.

**Pull request merged**